content/release-notes/2022-10-07.md

```diff
@@ -0,0 +1,13 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
+- Pageserver: Initial support for online tenant rebalancing
+- Pageserver: Support for multiple postgres versions (14 and 15 for now)
+- Compute: Add `h3_pg` extension
+- Compute: Beta version of sequential scan page prefetch
```

Comment:
```suggestion
- Compute: Introduced a beta version of sequential scan page prefetch, which improves sequential scan performance.
```

Reply:
Is the above addition accurate?

Reply:
It is, thanks!

Reply:
experimental? early? not sure about beta.
also, is the prefetch enabled in production? this entry looks like it is. is it actually isn't smth like 'added support for experimental sequential scan prefetch implementation' may be more appropriate.

Reply:
i am also thinking about the the readers who aren't familiar with db perf tuning
are there any links we can add here to help them relate to sequential scan performance?

Reply:
How about: 
"Added support for a future implementation of sequential scan prefetch, which improves I/O performance for operations such as table scans."

Reply:
Revised as mentioned above.